Packs/CortexAttackSurfaceManagement/Scripts/SnmpDetection/SnmpDetection.py

```diff
@@ -0,0 +1,107 @@
+from CommonServerPython import *  # noqa: F401
+
+
+import ipaddress
+import socket
+
+
+def snmp_send(udp_socket, probe, host, port):
+    """Return the Snmp response
+
+        Parameters:
+            udp_socket (object): Python object of socket.
+            probe (str): String value of Snmp data payload.
+            host (str): String value of IP address.
+            port (int): Integer port number.
+
+        Returns:
+            Str: Returns the value of the Snmp response.
+    """
+
+    udp_socket.sendto(bytes.fromhex(probe), (host, port))
+    udp_socket.settimeout(3)
+    return udp_socket.recvfrom(1024)
+
+
+def snmp_v1(udp_socket, probe, host, port):
```

Comment:
why we need 3 methods?
snmp_v1, snmp_v2 and snmp_v3 are having the same code

Reply:
Hello @adi88d, The reason why I have 3 methods, calling the same function is, it becomes easy to mock individual methods in my unit test cases, otherwise I won't be able to cover all the scenarios of test cases easily. The python file checks to see if the SNMP is enabled or not and also gets the versions enabled. Having these methods individually setup, I am able to mock each SNMP version response individually and test it. Hope that clarifies. Thanks